odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -37,6 +37,7 @@
     # OCA/account-reconcile
     'account_set_reconcilable': 'account',
     # OCA/l10n-spain
+    'l10n_es_account_invoice_sequence': 'account',
```

Comment:
delete this, as is already merged in l10n_es

Reply:
I'm seeing, thanks.